code/script.js

```diff
@@ -1,51 +1,482 @@
+	// the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+	if (sender === "user") {
+		chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/my-user.png" alt="User" />  
       </section>
-    `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
+    `;
+		// the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+	} else if (sender === "bot") {
+		chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/my-bot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight;
-}
-
-// Starts here
-const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
-}
+    `;
+	}
+	//To make the chat scroll down when there are new messages
+	chat.scrollTop = chat.scrollHeight;
+};
+
+//A function to count the questions and send to the next to next question based on that counting
+const nextQuestion = (message) => {
+	console.log("questionCounter", questionCounter);
+
+	if (questionCounter === 1) {
+		//user message will be display in the chat bubble (makes code shorter)
+		userReply(message);
+		//will display the user's name as a response to the helloUser function
+		input.value = "";
+		//Disables the function btnRemoveOnClick (in the end of the document) as I don't want the inputWrapper to be cleared after helloUser
+		btnRemoveOnClick(false);
+		setTimeout(() => ecoAnxietyOptions(message), 1000);
+	} else if (questionCounter === 2) {
+		userReply(message);
+		setTimeout(() => rateAnxiety(message), 1000);
+	} else if (questionCounter === 3) {
+		userReply(message);
+		setTimeout(() => nextStepOptions(message), 1000);
+	} else if (questionCounter === 4) {
+		userReply(message);
+		setTimeout(() => takeActionOptions(message), 1000);
+	} else if (questionCounter === 5) {
+		userReply(message);
+		setTimeout(() => finalStep(message), 1000);
+	} else if (questionCounter === 6) {
+		userReply(message);
+		setTimeout(() => anythingElse(message), 1000);
+	} else {
+		userReply(message);
+		setTimeout(thankYou, 1000);
+	}
```

Comment:
Would it be possible to place the setTimeout connected to another bit of code where it does not need to repeated, since every question is set to 1000?